src/azure_cosmosdb/cosmosdb_data_mappings.bal

```diff
@@ -0,0 +1,36 @@
+import ballerina/http;
+
+isolated function mapParametersToHeaderType(string httpVerb, string url) returns RequestHeaderParameters {
+    RequestHeaderParameters params = {};
+    params.verb = httpVerb;
+    params.resourceType = getResourceType(url);
+    params.resourceId = getResourceId(url);
+    return params;
+}
+
+isolated function mapResponseHeadersToObject(http:Response|http:ClientError httpResponse) returns @tainted Headers|error{
```

Comment:
Shall we add a space before the open curly bracket?
Please check this with all the functions.